pkg/kubectl/cmd/logs.go

```diff
@@ -199,7 +225,40 @@ func (o LogsOptions) Validate() error {
 
 // RunLogs retrieves a pod log
 func (o LogsOptions) RunLogs() (int64, error) {
-	req, err := o.LogsForObject(o.Object, o.Options)
+	switch t := o.Object.(type) {
```

Comment:
It can move there but I am now I have to build and return a fake Request since it would need to chain all the Pod logs in some manner.